src/app/pages/form/form.component.html

```diff
@@ -1 +1,67 @@
-<p>The Form Component</p>
\ No newline at end of file
+<p-card
+  styleClass="user-form"
+  class="flex aligh-items-center justify-content-center"
+>
+  <form [formGroup]="userForm" class="grid">
+    <div class="field col-6">
+      <label for="location" class="block">Location</label>
+      <p-dropdown
+        [options]="citiesFrom"
+        optionLabel="name"
+        [showClear]="true"
+        placeholder="Select a City"
+        formControlName="selectedCity"
+      ></p-dropdown>
+    </div>
+
+    <div class="field col-6">
+      <label for="destination" class="block">Destination</label>
+      <p-dropdown
+        [options]="citiesTo"
+        optionLabel="name"
+        [showClear]="true"
+        placeholder="Select a City"
+        formControlName="selectedCity"
+      ></p-dropdown>
+    </div>
+
+    <div class="">
+      <label for="departureDate" class="block col-3">Departure</label>
+      <!-- inline grid justify content space around -->
+      <p-calendar
```

Comment:
dobrze by było dodać do tej kontrolki `[showIcon]="true"` -> wtedy pokaże nam się ikona kalendarza w inpucie. To samo zastosować w niżej położonym `p-calendar`